### PR TITLE
add support for cli commands and error checking for invalid directories

### PR DESCRIPTION
**Pull request by mwimpelberg28**

### CODE REVIEW

File: file_renamer.py [hunk 1]
```diff
@@ -4,12 +4,30 @@
...
+parser = argparse.ArgumentParser(description='parse arguments from cli')
 
-# substring/regex replacements rules will be built here
-replacements = {}
+parser.add_argument('--target-path', action="store", dest='target_path', default="null")
+parser.add_argument('--name', action="store", dest='name', default="null")
+parser.add_argument('--replacement', action="store", dest='replacement', default="null")
+args = parser.parse_args()
+target_path = args.target_path
+name = args.name
```
Comment (dblinkhorn) [inline file_renamer.py:17]: can you rename this variable to "string" so it's clear to a user that it doesn't have to be the entire filename, but can just be a part of a filename they want to be replaced

**Review by dblinkhorn — CHANGES_REQUESTED** [review]
Should be able to fix the UI/CLI current incompatibility by wrapping all the the new parser code inside an `if __name__ == '__main__':` block.

File: file_renamer.py (hunk 1 above)
Comment (mwimpelberg28) [reply file_renamer.py:17]: I suppose I could do this for you good sir

**Review by dblinkhorn — APPROVED** [review]
Merging. Will push another commit fixing the incompatibility between CLI and UI code.

**Pull request merged**